src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+* {
```

Comment:
Don't use ``*`` selector (it impacts performance)
Better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_stars/assets/54860641/9c30f12d-5297-4da5-83bf-6bac47f55a75)